web/js/toggle_transliterated_string.js

```diff
@@ -0,0 +1,17 @@
+$(document).ready(function() {
+    $('.transliterate_button').click(function(event) {
+        var $transliterated = $('#transliterate_' + event.target.getAttribute('data-transliterated-id'));
+        var $normal = $('#string_' + event.target.getAttribute('data-transliterated-id'));
+        event.preventDefault();
+        var id = event.target.getAttribute('data-transliterated-id');
```

Comment:
Let's move this at the beginning and use the variable instead of calling getAttribute 3 times